server/sandbox.go

```diff
@@ -172,6 +172,7 @@ func (s *Server) RunPodSandbox(ctx context.Context, req *pb.RunPodSandboxRequest
 		if err != nil {
 			return nil, err
 		}
+		g.SetProcessSelinuxLabel(processLabel)
 	}
```

Comment:
@rhatdan do we need `label.ReserveLabel` here right? is there any way to release the label though? like on container's delete